tests/test.purchase.jsx

```diff
@@ -68,6 +73,14 @@ describe('Purchase', function() {
 
   it('should render a card entry form', function() {
     var View = mountView();
+
+    store.dispatch({
+      type: actionTypes.RX_BRAINTREE_TOKEN,
+      user: Object.assign({}, defaultUser, {
+        braintreeToken: 'braintree-token',
+      }),
```

Comment:
this isn't actually what the token action does. It only dispatches a token, not previous user data.